snap/snapcraft.yaml

```diff
@@ -126,12 +128,16 @@ parts:
   packages-deb:
     plugin: nil
     stage-packages:
-      - mysql-server-8.0=8.0.39-0ubuntu0.22.04.1
-      - mysql-router=8.0.39-0ubuntu0.22.04.1
+      - mysql-server-8.0=8.0.40-0ubuntu0.22.04.1
+      - mysql-router=8.0.40-0ubuntu0.22.04.1
       - mysql-shell=8.0.38+dfsg-0ubuntu0.22.04.1~ppa2
       - prometheus-mysqld-exporter=0.14.0-0ubuntu0.22.04.1~ppa2
       - prometheus-mysqlrouter-exporter=5.0.1-0ubuntu0.22.04.1~ppa1
       - percona-xtrabackup=8.0.35-31-0ubuntu0.22.04.1~ppa3
+      - mysql-audit-log-filter-plugin=8.0.39+dfsg-0ubuntu0.22.04.1~ppa6
+      - mysql-audit-log-plugin=8.0.39+dfsg-0ubuntu0.22.04.1~ppa6
+      - mysql-auth-ldap-plugin=8.0.39+dfsg-0ubuntu0.22.04.1~ppa6
```

Comment:
AFAIK we are not yet planning to integrate MySQL with LDAP, is this plugin here on purpose? If so, is it just to leave the plugin better prepare for the future?